cfme/services/catalogs/catalog_item.py

```diff
@@ -170,6 +171,9 @@ def create(self):
             tabstrip.select_tab("Catalog")
             template = template_select_form.template_table.find_row_by_cells({
                 'Name': self.catalog_name,
+                # HACK to workaround BZ1390209
+                # revert back to Provider once it's fixed
+                'Deprecated' if BZ(1390209, forced_streams=["5.7", "upstream"]).blocks else
```

Comment:
Looks like bugzilla entries are cached, so I guess they might be used like this?